docs/pages/connect-your-client/gui-clients.mdx

```diff
@@ -28,57 +26,93 @@ work with Teleport.
   [guides](../enroll-resources/database-access/guides/guides.mdx) for how to set up the Teleport
   Database Service for your database.
 
-### Get connection information
+## How GUI clients access Teleport-protected databases
+
+In a typical setup, clients accessing a database protected by Teleport send
+traffic to the Teleport Proxy Service in the database's native protocol, and the
+Proxy Service forwards the traffic to and from the protected database. End users
+use TLS certificates to authenticate with Teleport-protected databases. 
+
+GUI clients need to present a certificate to the Teleport Database Service and
+check the database server certificate issued by a Teleport-protected database.
+There are three ways to do this:
+
+- **Authenticated tunnel (recommended):** The local proxy handles loading the
+  Teleport user's client certificate and checking the certificate issued by the
+  Teleport-protected database. The connection remains authenticated, and clients
+  only need to forward database protocol messages to it.
+- **Unauthenticated local proxy:** The local GUI client needs to load the
```

Comment:
```suggestion
- **Unauthenticated local proxy:** The GUI client needs to load the
```

I think that adjective might be unnecessary since there are no non-local GUI clients involved.